_api-reference/index-apis/rollover.md

```diff
@@ -0,0 +1,196 @@
+
+The object body contains the following optional options for the alias:
+
+Parameter | Type | Description
+:--- | :--- | :---
+`filter` | Query DSL object | The query that limits documents the alias can access.
+`index_routing` | String | The value which routes indexing operations to a specific shard. When specified, overwrites the `routing` value for indexing operations.
+`is_hidden` | Boolean | When `true`, the alias is hidden. Default is false. All indexes for the alias must have matching values for this setting.
+`is_write_index` | Boolean | When `true`, the index is the [write index] for the alias. Default is `false`.
+`routing` | String | The value used to route index and search operations to a specific shard.
+`search_routing` | String | The value used to write specifically search operations to a specific shard. When specified, this option overwrites the `routing` value for search operations.
+
+### `mappings`
+
+The field mappings that exist in the index. For more information, see [Mappings and field types](https://opensearch.org/docs/latest/field-types/). Optional.
+
+### `conditions`
+
+The conditions parameter is an optional object that allows you to define specific criteria for triggering the rollover operation. When provided, OpenSearch only runs the rollover if the current index satisfies one or more of the specified conditions. However, if this parameter is omitted, OpenSerch will perform the rollover unconditionally, without any prerequisite conditions.
+
+The object body supports the following options.
+
+Parameter | Type | Description 
+:--- | :--- | :--- 
+| `max_age` | Time units | Triggers a rollover after the maximum elapsed time from index creation is reached. The elapsed time is always calculated since the index creation time, even if the index origination date is configured to a custom date, such as when using the `index.lifecycle.parse_origination_date` or `index.lifecycle.origination_date` settings. Optional. |
+`max_docs` | Integer | Triggers a rollover after the specified maximum number of documents is reached. Documents added since the last refresh are not included in the document count. The document count does not include documents in replica shards. Optional 
+`max_size` | Byte units  | Triggers rollover when the index reaches a certain size. This is the total size of all primary shards in the index. Replicas are not counted toward the maximum index size. To see the current index size, use the `_cat indices` API. The `pri.store.size` value shows the combined size of all primary shards. Optional.
+`max_primary_shard_size` | Byte units  | Triggers rollover when the largest primary shard in the index reaches a certain size. This is the maximum size of the primary shards in the index. As with `max_size`, replicas are ignored. To see the current shard size, use the `_cat shards` API. The `store` value shows the size of each shard, and `prirep` indicates whether a shard is a primary (`p`) or a replica (`r`). Optional.
```

Comment:
```suggestion
`max_primary_shard_size` | Byte units  | Triggers rollover when the largest primary shard reaches a specified size, ignoring replicas. Use the `_cat shards` API to see the current shard sizes (`store` value) and identify whether shard is primary (`p`) or replica (`r`). Optional.
```

Reply:
The `prirep` option is missing from the edit.